rust/agama-lib/src/bootloader/client/bootloader.rs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) [2024] SUSE LLC
```

Comment:
For consistency, I would follow the same approach as the other modules: `http_client.rs` and `client.rs` inside of `bootloader/` (or `http_client.rs` and `dbus_client.rs`).

Reply:
yeah, this was copy from previous location under storage directory. Will fix.